index.js

```diff
@@ -738,9 +756,42 @@ class BitbucketScm extends Scm {
      * @param  {Response}    Object          Object containing stats for the scm
      */
     stats() {
-        return this.breaker.stats();
+        const scmContexts = this._getScmContexts();
+        const scmContext = scmContexts[0];
+        const stats = this.breaker.stats();
+
+        return { [scmContext]: stats };
+    }
+
+   /**
+    * Get an array of scm context (e.g. bitbucket:bitbucket.org)
+    * @method getScmContexts
+    * @return {Array}
+    */
+    _getScmContexts() {
+        const contextName = [`bitbucket:${this.hostname}`];
+
+        return contextName;
     }
 
+   /**
+    * Determine if a scm module can handle the received webhook
+    * @method canHandleWebhook
+    * @param {Object}    headers     The request headers associated with the webhook payload
+    * @param {Object}    payload     The webhook payload received from the SCM service
+    * @return {Promise}
+    * */
+    _canHandleWebhook(headers, payload) {
+        return this._parseHook(headers, payload).then((parseResult) => {
+            if (parseResult === null) {
+                return Promise.resolve(false);
+            }
+
+            const [, checkoutUrlHost] = parseResult.checkoutUrl.split('@');
+
+            return Promise.resolve(checkoutUrlHost.startsWith(this.hostname));
+        });
```

Comment:
Do we need a catch here in case `_parseHook` fails?
https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L822-L824